docs/tutorials/lander.md

```diff
@@ -0,0 +1,340 @@
+```
+
+## Step 6 @fullscreen
+
+In ``||variables:Variables||`` grab a ``||variables:set to||`` block and drag it into on start. Change the variable name to angle and then drag a text box from ``||text:Text||`` and drag it into the value. Set the value to straight.
+
+```blocks
+let angle = ""
+angle = "straight"
+```
+
+## Step 7 @fullscreen
+
+Next drag an ``||controller:on A button pressed||`` from ``||controller:Controller||`` and drag it into your workspace. From ``||logic:Logic||`` grab an ``||logic:if then else||`` statement and drag it into ``||controller:on A button pressed||``. Then click the plus arrow. Grab a ``||logic:0=0||`` block from from ``||logic:Logic||`` and place two of them in the if else if sections. For both equal statements set the first value to ``||variables:angle||`` and for the first set the second field to ``||text:" "||`` of straight and the second to  ``||text:" "||`` of left. 
+
+
+## Step 8 @fullscreen
+
+Now from ``||sprites:Sprite||`` grab a ``||sprites:set sprite velocity to vx vy||`` and place it in all three of the sections. Set the sprite variable for all three to Lander. In the first set vx to **0** and vy to **-20**. In the second set vx to **-6** and vy to **-20**. Then in the last set set the vx to **6** and vy to **-20**.
+
+```blocks
+let Lander: Sprite = null
+let angle = "straight"
+controller.A.onEvent(ControllerButtonEvent.Pressed, function () {
+    if (angle == "straight") {
+        Lander.setVelocity(0, -20)
+    } else if (angle == "left") {
+        Lander.setVelocity(-1 * Math.sqrt(20), -20)
+    } else {
+        Lander.setVelocity(1 * Math.sqrt(20), -20)
+    }
+})
+```
+
+## Step 9 @fullscreen
+
+When the sprite hits the lunar surface it should die. So grab from ``||scene:Scene||`` grab a ``||scene: on sprite of kind Player hits wall||`` and place it in the workspace. Make sure the color is set to the lunar surface, white. From ``||sprites:Sprite||`` grab a ``||sprites:destroy sprite||``. Change the sprite to Lander and select the plus arrow. Set the effect to fire and the time to 500 ms. From ``||game:Game||`` grab a ``||game:game over lose||`` and click the plus arrow and set the effect to dissolve.
+
+```blocks
+let Lander: Sprite = null
+scene.onHitTile(SpriteKind.Player, 1, function (sprite) {
+    Lander.destroy(effects.fire, 500)
+    game.over(false, effects.dissolve)
+})
+```
+
+## Step 10 @fullscreen
+
+Next grab from ``||scene:Scene||`` a ``||scene:on sprite of kind Player hits wall||`` and drag it into the workspace. Make sure to set the color to the landing pad, orange. Then grab from ``||logic:if then else||`` and drag it into the block. Again from ``||logic:Logic||`` grab ``||logic:0=0||`` and drag it into the if statement. Grab the ``||variables:angle||`` from the ``||variables:Variable||`` drawer and drag it into the first field of the equal statement. Grab a ``||text:" "||`` from ``||text:Text||`` and place it in the second field and type straight. 
```

Comment:
hint snippet for this step?

Reply:
This is also the comment I made on step 7. The block segment isn't complete in this phase so I wasn't sure if it made sense to add in an incomplete hint snippet for this step.

Reply:
Tutorials are incremental and we don't want to leave anyone guessing. Either have the whole thing or a smaller snippet of the blocks introduced in that step.